app/models/patient.rb

```diff
@@ -1267,13 +1267,18 @@ def monitoring_history_edit(history_data, diff_state)
     # NOTE: Attributes are sorted so that:
     # - case_status always comes before isolation, since a case_status change may trigger an isolation change from the front end
     # - continuous_exposure comes before last_date_of_exposure, since a continuous_exposure change may trigger an lde change from the front end
-    attribute_order = %i[case_status isolation continuous_exposure last_date_of_exposure]
+
+    # When closing a monitoree via updating their case_status, the `reason` field will be set on both History items
+    # Keep track of when monitoring has been updating, and then dont include 'reason' for case_status
```

Comment:
I just want to make sure I understand what you are trying to communitcate in this comment - Are you saying that since the `reason` field would usually be used for both the Monitoring and Case Status history items, you are using the `has_updated_monitoring` flag to ensure the `reason` field is only used for the Monitoring history item instead of both? I'm just a bit confused since the first line of this comment suggests to me that you want the `reason` field to be included on both items.

Reply:
I've re-worded this to be:
```    
# When closing a monitoree via updating their case_status, the `reason` field will (incorrectly) be set on both History items
# To prevent this, we want to keep track of when monitoring has been updating, and then not include 'reason' for case_status History item